libs/vkd3d/command.c

```diff
@@ -976,7 +992,7 @@ static CONST_VTBL struct ID3D12Fence1Vtbl d3d12_fence_vtbl =
     d3d12_fence_GetPrivateData,
     d3d12_fence_SetPrivateData,
     d3d12_fence_SetPrivateDataInterface,
-    d3d12_fence_SetName,
+    (void *)d3d12_object_SetName,
```

Comment:
This feels a bit gross. Casting non-fn-pointer to fn-pointer tends to trip warnings. Is there a more direct fn-pointer type we can cast to here?

Reply:
None that have a typedef and wouldn't be incredibly long.

On POSIX-compliant systems, `sizeof(void*) == sizeof(some func ptr)` and we aren't planning on supporting IA64 outside of that so we should be good here.

Reply:
Ok